### PR TITLE
edit profile

### PR DESCRIPTION
**Pull request by quyensama**
## Related Tickets:
- [#45918](https://edu-redmine.sun-asterisk.vn/issues/45918)

## WHAT this PR does?
- edit profile info
- change password

## Notes Impacted Areas:
- N/A.

## Attachments:
Screenshot from 2022-03-09 10-25-11

Screenshot from 2022-03-09 10-25-27

### CODE REVIEW

**Review by NamBZ — APPROVED** [review]

**Comment by quyensama** [comment]
ready

File: app/Http/Requests/Profile/UpdateRequest.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'name' => [ 'string', 'min:3', 'max:255'],
```
Comment (ghost) [inline app/Http/Requests/Profile/UpdateRequest.php]:
```suggestion
            'name' => ['string', 'min:3', 'max:255'],
```

**Comment by quyensama** [comment]
ready

**Comment by quyensama** [comment]
ready

File: database/migrations/2022_03_09_023638_add_address_into_users_table.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        if (Schema::hasColumn('address')) {
+            Schema::table('users', function (Blueprint $table) {
+                $table->dropColumn(['address']);
+            });
```
Comment (huyenttm-0901) [inline database/migrations/2022_03_09_023638_add_address_into_users_table.php]: e chạy thử hàm down() chưa? check hasColumn trong table chứ

**Comment by quyensama** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**